### ISSUE TITLE
Media Nodes not retaining Sort Order upon migration

### ISSUE THREAD

**Issue by JMMM77** [Jumoo/uSyncMigrations#121]
When I was migrating a view I noticed that the images were not displaying in the correct order, I looked at the media tab in the back office and saw that the folders and images were not in the same order either. When I sorted the folder that the code was using to display the images I saw that this was reflected on the front end and the images were now ordered correctly.

I could not find any obvious sort by property that I could use to sort it and it seems like the migrated media types were sorted based on id, but is there a way that I can run the uSync Migrations so that the media nodes keep the order that they were in originally?

Please let me know if there is anything else I can help with.

Before Migration:
![image](https://github.com/Jumoo/uSyncMigrations/assets/91952323/5c28ac0a-1775-4ca4-b277-d0ed6fc54caa)

After Migration:
![image](https://github.com/Jumoo/uSyncMigrations/assets/91952323/46cca092-3f5a-42ff-b969-49618f2ea828)

**Comment by KevinJump** [comment]
Yes - from memory (and i will have to go check). there is a sort order for media, but not all media views actually sort by it 🤨.

I think for example the tree might be sorted but the views are based on created/update date. - in full on uSync.Complete we do something to clever to sync things in created date order and then ensure sorting... so it might be something we can do here too.
